modules/admin_manual/pages/maintenance/enable_maintenance.adoc

```diff
@@ -8,18 +8,17 @@ Please see xref:configuration/server/occ_command.adoc[Using the occ Command] to
 new logins. This is the mode to use for upgrades. You must run `occ` as
 the HTTP user, like this example on Ubuntu Linux:
 
-....
-$ sudo -u www-data php occ maintenance:mode --on
-....
+[source,console]
+----
+sudo -u www-data php occ maintenance:mode --on
+----
 
-You may also put your server into this mode by editing
-config/config.php. Change `"maintenance" => false` to
-`"maintenance" => true`:
+You may also put your server into this mode by editing config/config.php. +
```

Comment:
Can we avoid this practice, pls?

Reply:
If you look to the rendered output before and after, you see the difference and it is much better readable now.